kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/containOnly.kt

```diff
@@ -0,0 +1,181 @@
+package io.kotest.matchers.collections
+
+import io.kotest.assertions.print.print
+import io.kotest.matchers.*
+import kotlin.jvm.JvmName
+import io.kotest.assertions.AssertionsConfig.maxCollectionPrintSize as printSize
+
+/**
+ * Assert that a collection contains only the given elements.
+ * For example,
+ *  listOf(1, 1, 2) shouldContainOnly listOf(1, 2)    // Assertion passes
+ *  listOf(1, 2, 1) shouldContainOnly listOf(1)       // Assertion fails
+ *  listOf(1) shouldContainOnly listOf(2, 2)          // Assertion fails
+ *
+ *  Note: Comparison is via the standard Java equals and hash code methods.
```

Comment:
We are in common kotlin land, so I think these statements are innacurate

Reply:
I think he's saying that the equals comes from the Kotlin common module not the JVM object superclass, since this isn't a JVM module.

But I don't think it matters too much we all know what it means.